### ISSUE TITLE
ArcticJS 2017 pre-announcement

### ISSUE THREAD

**Issue by watson** [adventure-js/arcticjs.club#21]
**Save these dates: December 16th + 17th**

We still need to verify that there isn't some other big conference in the city at that time, but if nothing derails this, then it's going to be those dates 😃

@mafintosh and I will probably fly up there on Monday the 11th of December and back on Monday the 18th. Alternatively from Friday the 15th to Friday the 22nd.

**Comment by yangwao** [comment]
I'm in, for 70%. ^_^      
34c3 will be ≈ 27/12-30/12.

**Comment by carlesba** [comment]
What companies fly there?

**Comment by watson** [comment]
@carlesba I'd recommend checking https://google.com/flights

Last time we found that there were a lot of money to be saved by buying one ticket to Oslo (the closest big hub) and then buy a separate ticket from there to Longyearbyen with Norwegian.

Norwegian fly in and out of Longyearbyen on Mondays and Fridays.

**Comment by despairblue** [comment]
Noted the dates. See you there!

**Comment by MylesBorins** [comment]
Noted :D

**Comment by feross** [comment]
🐻

**Comment by elsehow** [comment]
Is it alright to come if I've never attended before?

**Comment by feross** [comment]
@elsehow Yes! That is highly encouraged! 👍 👍 👍

**Comment by hackergrrl** [comment]
Do come, Nick!

**Comment by RichardLitt** [comment]
Any chance we can have a CFP? Scholarships? Bursaries? 

Should we book AirBnB's now?

**Comment by yangwao** [comment]
[From Vienna is price $274 ≈ 14/12/2016](https://www.google.com/flights/#search;f=VIE,XWW;t=LYR;d=2017-12-14;r=2018-01-08;tt=o;sel=VIEOSL0OS331-OSLTOS1SK4406-TOSLYR1SK4414)
![image](https://cloud.githubusercontent.com/assets/5887929/22971678/ca072c00-f376-11e6-9fec-089ab28c03c6.png)

**Comment by watson** [comment]
@RichardLitt I'm currently looking into whether or not we should reserve an entire hotel or if we should just tick with Airbnb. There's room for 16 people combined in the Airbnb's

**Comment by whyrusleeping** [comment]
@watson how many can the hotel handle? Wasnt it way up the road?

**Comment by watson** [comment]
@whyrusleeping depends on which hotel. There's around 7 hotels. Two are too far away.

**Comment by RichardLitt** [comment]
The AirBnB was certainly a nicer experience than I think a hotel would be. But I don't know if it was more expensive or not. 

There's a few hotels up near the Coffin mountain. Probably too far. Polar bear territory. :D

**Comment by despairblue** [comment]
Last time we've been [here](https://www.google.de/maps/place/Gjestehuset+102+(Guesthouse+102)/@78.2011119,15.5910914,16.25z/data=!4m5!3m4!1s0x0:0xea1853523288d47a!8m2!3d78.200834!4d15.5901213?hl=de) which was reasonable cheap, but it is half a walking hour away from the city center. 

I'd like to be more central next time, so if we could all stay in either the same hotel or in 2 big airbnbs that are close to each other, that would be awesome.

**Comment by watson** [comment]
New 2017 website is up: http://arcticjs.club/2017/ (I moved the old site to http://arcticjs.club/2016/)

**Comment by RichardLitt** [comment]
Looks good! Link to the last one somewhere, and to the GitHub source code?

**Comment by watson** [comment]
@RichardLitt I plan to update to something more substantial later. Just wanted to get the newsletter signup out. Other than that everyone if of course welcome to improve it 😃

**Comment by RichardLitt** [comment]
PRed.

**Comment by gsans** [comment]
First timer here. I'd like to join but Im not sure how the conference goes. Is there a CFP I could send a talk? What is the theme?

**Comment by julianduque** [comment]
looking forward to, If anyone plans to be in Norway (Oslo/Bergen) before we can plan something

**Comment by watson** [comment]
@gsans We'll open up for CFP soon. Sign up for the newsletter on https://arcticjs.club or keep an eye on the issues here on github to be notified. The theme is whatever you want to talk about 😃

**Comment by MylesBorins** [comment]
OH THERE IS A CFP! Is there a plan for scholarship tickets? LMK if you need
help with curation

**Comment by trygve-lie** [comment]
So; what if we try to pull this off a bit earlier? Lets say just after Web Rebels in Oslo (1-2 of June).

**Comment by watson** [comment]
@MylesBorins thanks 😃 Yes that was the plan (though it's still in the very early stages). What does scholarship tickets cover exactly?

@trygve-lie Interesting idea. We did choose December because it would be polar night at that time, it would give us more time to organise the whole thing, and because there wasn't that many conferences in December to compete with. But I get your point about having the two conferences closer together to allow participants to attend both on the same trip.

**Comment by MylesBorins** [comment]
Is this still likely to happen?

**Comment by RichardLitt** [comment]
AFAIK depends on @watson, who was looking into organizing and setting dates in Stone. 

@watson any way we can help?

**Comment by RichardLitt** [comment]
@watson Any chance we could do this in January, so it doesn't hit near Christmas flights? 11th to 18th works for me, though, which might be OK for flights. You booked your tickets yet?

**Comment by diffcunha** [comment]
Is this still going to happen? Yeah, I'm available to give some help if needed. @diasdavid what's you plan?

**Comment by yangwao** [comment]
Looks like radio silence from @watson, is anyone else in org team?

**Comment by RichardLitt** [comment]
"org team" is kind of a loose concept. Really, we've been pretty unfair in putting this on Watson. 

Yes, this can still happen. Obviously, it's not happening tomorrow (unless someone is going that I don't know about?)

Our best bet would be to do one of two things: post pone for next year, or gauge interest in having it in January again. 

React to this comment with a 🎉 if you want it to happen in January or February and are interested in setting up dates. React with a 👎 if you'd like it to happen next winter, or next summer at some point so that it's not entirely dark again.

**Comment by RichardLitt** [comment]
Looks like we have some people who would be able to make it in January and February. I'd suggest that you set up a separate issue to plan for that time - if four of you go, it's probable that some others will come, too - which would mean around the same amount as last time. (I won't be able to make it, and don't want to be involved in the planning for that reason, but can certainly help advise). 

For now, closing this, as it is not happening in 2017.
